guides/advanced_guides/snapshots_and_dumps.md

```diff
@@ -96,7 +106,7 @@ After your dump creation process is done, the dump file is created and added in
 Once you have exported a dump, which is a `.dump` file, you are now able to use that dump to launch MeiliSearch. As the data contained in the dump needs to be indexed, the process will take some time to complete. Only when the dump has been fully imported will the MeiliSearch server start, after which you can begin searching through your data.
 
 ```bash
-./meilisearch --import-dump /dumps/12345678.dump
+./meilisearch --import-dump /dumps/20200813-042312213.dump
 ```
 
 Because the indexation is the same process as when the documents are initially added to MeiliSearch, it is still a good practice to index documents in batches if the dataset is to big. The size of the batches are based on your dataset size and on your memory capacity.
```

Comment:
```suggestion
Because importing a dump is the same process as when documents are initially indexed by MeiliSearch, it can require some time and memory. If your dataset is very large, it is good practice to index documents in larger batches. This will speed up the indexing process at the cost of memory.
```